### ISSUE TITLE
Support web platform (react-native-web)

### ISSUE THREAD

**Issue by brunolemos** [fdnhkj/react-native-shared-code#5]
I tried just adding `web` to the platform list, but there were some exceptions like module XXX does not exist.

**Comment by fdnhkj** [comment]
Good point! Could you please share with me a react-native-web project giving such exceptions so I can start investigating ?
